library/Vanilla/UploadedFile.php

```diff
@@ -295,6 +312,40 @@ public function moveTo($targetPath) {
         $this->moved = true;
     }
 
+    /**
+     * Get max image upload height
+     * If $this->maxImageHeight is set to NO_IMAGE_DIMENSIONS_LIMIT, fallback to return MAX_IMAGE_HEIGHT
+     *
+     * @return ?int
+     */
+    public function getMaxImageHeight(): ?int {
+        switch($this->maxImageHeight) {
+            case null:
```

Comment:
Have you tested this? I am pretty sure that a switch is the same as `==` rather than `===`.